### PR TITLE
feat: delete user endpoint

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part pf #1099 
Added endpoint to remove user

### CODE REVIEW

File: src/main/java/com/artipie/api/UsersRest.java [hunk 1]
```diff
@@ -51,6 +52,24 @@ public void init(final RouterBuilder rbr) {
...
+    }
+
+    /**
+     * Removes user.
+     * @param context Request context
+     */
+    private void deleteUser(final RoutingContext context) {
+        try {
+            this.users.remove(context.pathParam(RepositoryName.UNAME));
+            this.cache.invalidateAll();
```
Comment (andpopov) [inline src/main/java/com/artipie/api/UsersRest.java:67]: Can we to narrow the 'try'-block to catch exactly exception rom 'users.remove(...)'?
I mean:
        try {
            this.users.remove(context.pathParam(RepositoryName.UNAME));
        } catch (final IllegalStateException err) {
            Logger.error(this, err.getMessage());
            context.response().setStatusCode(HttpStatus.NOT_FOUND_404).end();
        }
        this.cache.invalidateAll();
        context.response().setStatusCode(HttpStatus.OK_200).end();

**Review by andpopov — CHANGES_REQUESTED** [review]

**Comment by olenagerasimova** [comment]
@andpopov thanks, corrected

**Review by dgarus — APPROVED** [review]
@olenagerasimova thanks!

File: src/main/java/com/artipie/api/UsersRest.java (hunk 1 above)
Comment (andpopov) [reply src/main/java/com/artipie/api/UsersRest.java]: @olenagerasimova thanks!

**Review by andpopov — APPROVED** [review]

**Pull request merged**